static/skin/viewer.js

```diff
@@ -262,10 +263,8 @@ function onClickEvent(e) {
   const target = matchingAncestorElement(e.target, iframeDocument, "a");
   if (target !== null && "href" in target) {
     if ( isExternalUrl(target.href) ) {
-      target.setAttribute("target", "_top");
-      if ( viewerSettings.linkBlockingEnabled ) {
-        return blockLink(target);
-      }
+      target.setAttribute("href", blockLink(target.href));
```

Comment:
Why do we need to change the target's href ?

It will be the viewer which will change the window location.

Reply:
This is required if we also want to block links opened in a new tab/window.